build-config/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -184,26 +184,49 @@
   <module name="RegexpHeader">
     <property name="header" value="^/\*\n^ \* Copyright \(C\) (2009|20[12][0-9]) - present by OpenGamma Inc\. and the OpenGamma group of companies\n^ \*\n^ \* Please see distribution for license\.\n^ \*\/"/>
     <property name="fileExtensions" value="java,proto"/>
-    <property name="severity" value="error"/>
+  </module>
+  <module name="RegexpSingleline">
+    <!-- Disallow the @author tag in Javadoc. -->
+    <!-- (Not currently implemented explicitly in Checkstyle: https://github.com/checkstyle/checkstyle/issues/2506) -->
+    <property name="format" value="^\s*\*\s*@author"/>
+    <property name="message" value="Javadoc has illegal ''author'' tag."/>
+    <property name="fileExtensions" value="java"/>
+  </module>
+  <module name="RegexpSingleline">
+    <!-- (?!\h*\/\/|\h*\/\*\*|\h*\*\/|\h*\*) ignores lines beginning with comments (javadoc or plain) -->
+    <property name="format" value="^(?!\h*\/\/|\h*\/\*\*|\h*\*\/|\h*\*).*\([a-zA-Z\h,]+$"/>
+    <property name="message" value="Multiple arguments must be placed on their own line"/>
```

Comment:
The rule is not that multiple args must each be on their own line. If they fit on one line, that is allowed, and there are cases where it is better to group args, eg `ImmutableMap.of()`

Reply:
A misquoting on my part. Have updated it to read

> When split across multiple lines, each argument must be placed on its own line